pkg/lib/operation/project/remote/file/upload/operation.go

```diff
@@ -32,42 +34,101 @@ func Run(ctx context.Context, o Options, d dependencies) (err error) {
 	ctx, span := d.Tracer().Start(ctx, "kac.lib.operation.project.remote.file.upload")
 	defer telemetry.EndSpan(span, &err)
 
-	var reader io.Reader
-	var bar *progressbar.ProgressBar
-	if o.Input == "-" {
-		reader = bufio.NewReader(os.Stdin)
-	} else {
-		file, err := os.Open(o.Input) // nolint: forbidigo
+	opts := make([]keboola.CreateFileOption, 0)
+	if len(o.Tags) > 0 {
+		opts = append(opts, keboola.WithTags(o.Tags...))
+	}
+
+	// Check files existence
+	var fr *os.File
+	var fi os.FileInfo
+	if o.Input != "-" {
+		fr, err = os.Open(o.Input) // nolint: forbidigo
```

Comment:
Proč používáš `os` místo `filesystem`?

Reply:
Nó, je to filesystem projektovýho folderu a tohle je remote command který na něm nijak nezávisí. A myslel jsem si teda že je to nahraný do paměti, tak přes něj nepůjde volat soubory mimo aktuální adresář, ale teď se dívám že asi ne. Zkusím to teda použít.

Reply:
Jo, fakt to nejde.
![CleanShot 2023-03-22 at 15 00 57](https://user-images.githubusercontent.com/215660/226928329-88795fd3-fa62-440c-b4c2-87f54e900ea3.jpg)